### PR TITLE
ltv bq scripts and notebooks with adwords activation

### PR DESCRIPTION
**Pull request by m-mayran**
Folder to run LTV
- Notebooks with example script for Adwords
- BQ Scripting and Stored Procedures

### CODE REVIEW

File: retail/ltv/bqml/README.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Activate on LTV predictions.
```
Comment (polong-lin) [inline retail/ltv/bqml/README.md:1]: Remove period at end of title

File: retail/ltv/bqml/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+There are two main sets of files:
+
+**[1. ./notebooks](./notebooks)**
+
+You can use the notebook in this folder to manually run the flow using example datasets. You can also used your own data.
+
+**[2. ./scripts](./scripts)**
+
+The scripts in this folder facilitate automation through BigQuery scripting, BigQuery stored procedures and bash scripting. Scripts use statements from the notebook to:
```
Comment (polong-lin) [inline retail/ltv/bqml/README.md:38]: Suggestion to add links to "BigQuery scripting, BigQuery stored procedures":

[BigQuery scripting](https://cloud.google.com/bigquery/docs/reference/standard-sql/scripting), [BigQuery stored procedures](https://cloud.google.com/blog/products/data-analytics/command-and-control-now-easier-in-bigquery-with-scripting-and-stored-procedures)

File: retail/ltv/bqml/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+1. Train and use model to predict LTV.
+1. Extract emails of the top LTV customers.
+
+*Note: For production use cases, you can reuse the SQL statements from the scripts folder in pipeline tools such as Kubeflow Pipelines or Cloud Composer.*
+
+The scripts assume that you already have the sales and crm datasets stored in BigQuery.
+
+## Recommended flow
+
+1. Do research in the Notebook.
```
Comment (polong-lin) [inline retail/ltv/bqml/README.md:49]: Change "Notebook" --> "Jupyter Notebook"
with an embedded link to the .ipynb file

File: retail/ltv/bqml/README.md (hunk 1 above)
Comment (m-mayran) [reply retail/ltv/bqml/README.md]: Fixed in d5ee0e2255d617ee1ea5b041f206bee0d41e5906

File: retail/ltv/bqml/README.md (hunk 3 above)
Comment (m-mayran) [reply retail/ltv/bqml/README.md]: Fixed in d5ee0e2255d617ee1ea5b041f206bee0d41e5906

File: retail/ltv/bqml/README.md (hunk 2 above)
Comment (m-mayran) [reply retail/ltv/bqml/README.md]: Fixed in d5ee0e2255d617ee1ea5b041f206bee0d41e5906

**Pull request merged**